ui_components.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'rails-assets-bootstrap-chosen'
   s.add_dependency 'rails-assets-marked'
   s.add_dependency 'rails-assets-ajax-chosen'
+  s.add_dependency 'rails-assets-highlightjs'
```

Comment:
Why is this a dependency of the gem but `bootstrap-sass` is only defined in the dummy app Gemfile?